### PR TITLE
Solving Issue Link for Testing Brokenon contributor cheatsheet#6872

### PR DESCRIPTION
**Pull request by Pro-Coder04**
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by palnabarun** [comment]
Please sign the CLA.

File: contributors/guide/contributor-cheatsheet/README.md [hunk 1]
```diff
@@ -422,3 +422,4 @@ git push --force
 [approve]: https://prow.k8s.io/command-help#approve
 [GitHub Administration Team]: /github-management#github-administration-team
 [Kubernetes Patch Release]: https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md
+[Testing locally]:https://jamesdefabia.github.io/docs/getting-started-guides/docker/
```
Comment (palnabarun) [inline contributors/guide/contributor-cheatsheet/README.md:425]: The reference exists above as well. We shouldn't define it again and instead fix the existing reference to the correct link.

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: contributors/guide/contributor-cheatsheet/README.md (hunk 1 above)
Comment (Pro-Coder04) [reply contributors/guide/contributor-cheatsheet/README.md:425]: What should I do next?
Could You suggest me some fixes since I'm new to open source

Comment (Pro-Coder04) [reply contributors/guide/contributor-cheatsheet/README.md]: I have updated the PR as per your suggestions @palnabarun. could you please review?

**Comment by palnabarun** [comment]
A closer fix is being worked on at #6879.

Hence closing this. Thank you for taking a dig at this!

/close

**Comment by Pro-Coder04** [comment]
@palnabarun Thanks for guiding me through my first contribution for kubernetes, though it wasn't accepted
If you find anymore such issues please do assign them to me.

**Pull request closed without merging**